test/scripts/imgtestlib.py

```diff
@@ -254,3 +291,32 @@ def clargs():
                         help="architecture to generate configs for (defaults to host architecture)")
 
     return parser
+
+
+def read_osrelease():
+    """Read Operating System Information from `os-release`
+
+    This creates a dictionary with information describing the running operating system. It reads the information from
+    the path array provided as `paths`.  The first available file takes precedence. It must be formatted according to
+    the rules in `os-release(5)`.
+    """
+    osrelease = {}
+
+    with open(OS_RELEASE_FILE, encoding="utf8") as orf:
```

Comment:
Another way to do this (possibly overkill?) https://github.com/weldr/lorax/blob/master/src/pylorax/sysutils.py#L115

Reply:
Maybe a little overkill yeah.
Or we can just copy the example from [the manpage](https://www.freedesktop.org/software/systemd/man/latest/os-release.html) :smile: